app/controllers/api/v2/anime_controller.rb

```diff
@@ -2,6 +2,21 @@ module Api::V2
   class AnimeController < ApiController
     caches_action :show, expires_in: 1.hour
 
+    def index
+      anime = Anime.preload(:genres)
+
+      if params.key? :filter
```

Comment:
[bbatsov is vague](https://github.com/bbatsov/ruby-style-guide#no-dsl-parens), but seems to lean towards including parens.  GitHub's style guide says nothing on the matter.

From a quick run through `rubocop`, I'm gonna side with GitHub and say "fuck it who cares" for predicate methods